bin/picca_nersc_submit

```diff
@@ -0,0 +1,435 @@
+    fout = open(fbatch,"w")
+    fout.write(srun)
+    fout.close()
+
+    return fbatch
+
+def do_ini(outdir, cf_file):
+    '''
+    Writes the .ini files to control the fits
+    Input:
+        - outdir (string): output directory of the fits
+        - cf_file (string): path to the input file for the fitter
+    Returns:
+        The (basename) of the chi2.ini file
+    '''
+    fout = open(outdir+"/"+cf_file.replace(".fits",".ini"),"w")
+    fout.write("[data]\n")
+    fout.write("name = {}\n".format(basename(cf_file).replace(".fits","")))
+    fout.write("tracer1 = LYA\n")
+    fout.write("tracer1-type = continuous\n")
+    if "xcf" in cf_file:
+        fout.write("tracer2 = QSO\n")
+        fout.write("tracer2-type = continuous\n")
+    else:
+        fout.write("tracer2 = LYA\n")
+        fout.write("tracer2-type = continuous\n")
+    fout.write("filename = {}\n".format(cf_file))
+    fout.write("ell-max = 6\n")
+
+    fout.write("[cuts]\n")
+    fout.write("rp-min = -200\n")
+    fout.write("rp-max = 200\n")
+
+    fout.write("rt-min = 0\n")
+    fout.write("rt-max = 200\n")
+
+    fout.write("r-min = 10\n")
+    fout.write("r-max = 180\n")
+
+    fout.write("mu-min = -1\n")
+    fout.write("mu-max = 1\n")
+
+    fout.write("[model]\n")
+    fout.write("model-pk = pk_kaiser\n")
+    fout.write("z evol LYA = bias_vs_z_std\n")
+    fout.write("growth function = growth_factor_no_de\n")
+    if "xcf" in cf_file:
+        fout.write("model-xi = xi_drp\n")
+        fout.write("z evol QSO = qso_bias_vs_z_croom\n")
+    else:
+        fout.write("model-xi = xi\n")
+
+
+    fout.write("[parameters]\n")
+
+    fout.write("ap = 1. 0.1 0.5 1.5 free\n")
+    fout.write("at = 1. 0.1 0.5 1.5 free\n")
+    fout.write("bias_LYA = -0.17 0.017 None None free\n")
+    fout.write("beta_LYA = 1. 0.1 None None free\n")
+    fout.write("alpha_LYA = 2.9 0.1 None None fixed\n")
+    if "xcf" in cf_file:
+        fout.write("bias_QSO = 1. 0.1 None None fixed\n")
+        fout.write("beta_QSO = 0.3 0.1 None None free\n")
+
+    fout.write("growth_rate = 0.962524 0.1 None None fixed\n")
+
+    fout.write("sigmaNL_per = 3.24 0.1 None None fixed\n")
+    fout.write("1+f = 1.966 0.1 None None fixed\n")
+
+    fout.write("par binsize {} = 4. 0.4 None None free\n".format(cf_file.replace(".fits","")))
+    fout.write("per binsize {} = 4. 0.4 None None free\n".format(cf_file.replace(".fits","")))
+
+    fout.write("bao_amp = 1. 0.1 None None fixed\n")
+    if "xcf" in cf_file:
+        fout.write("drp = 0. 0.1 None None free\n")
+        fout.write("croom_par0 = 0.53 0.1 None None fixed\n")
+        fout.write("croom_par1 = 0.289 0.1 None None fixed\n")
+    fout.close()
+
+    chi2_ini = outdir+"/chi2_{}".format(cf_file.replace(".fits",".ini"))
+    fout = open(chi2_ini,"w")
+    fout.write("[data sets]\n")
+    fout.write("zeff = 2.310\n")
+    fout.write("ini files = {}\n".format(cf_file.replace(".fits",".ini")))
+
+    fout.write("[fiducial]\n")
```

Comment:
You can remove these two lines. picca will take the local power spectrum by default